include/openssl/dh.h

```diff
@@ -92,7 +92,11 @@ int EVP_PKEY_CTX_get0_dh_kdf_ukm(EVP_PKEY_CTX *ctx, unsigned char **ukm);
 #  include <openssl/dherr.h>
 
 #  ifndef OPENSSL_DH_MAX_MODULUS_BITS
-#   define OPENSSL_DH_MAX_MODULUS_BITS    10000
+#   define OPENSSL_DH_MAX_MODULUS_BITS        10000
+#  endif
+
+#  ifndef OPENSSL_DH_MAX_CHECK_MODULUS_BITS
+#   define OPENSSL_DH_CHECK_MAX_MODULUS_BITS  32768
```

Comment:
Should it be `OPENSSL_DH_MAX_CHECK_MODULUS_BITS` or `OPENSSL_DH_CHECK_MAX_MODULUS_BITS`?

Reply:
Good catch! Fixed.